pkg/agent/iptables/iptables_windows.go

```diff
@@ -0,0 +1,30 @@
+package iptables
```

Comment:
I would agree with what @tnqn said - do not create dummy funcs if they are not needed for an OS.

Reply:
@jianjuns  I don't think we could remove iptables_windows.go(or traffic_filter_windows.go) even if we define an interface. Since "iptables" is a Linux specific library, all functions calling this library should be placed in file with _linux suffix, otherwise it should break compiling on Windows. If so, function "allowNodeToPeerCIDRTraffic" must have separate implementations with different OS type, and the WIndows implementation might be an empty function. So do other functions we plan to in the interface.

Reply:
No, I am not saying we can remove traffic_filter_windows.go. I am just saying we should not call it iptables_windows.go, and if we have an interface, it just includes a single func of allowNodeToPeerCIDRTraffic().

Reply:
traffic_filter_windows.go should include the implementation of allowNodeToPeerCIDRTraffic().

Reply:
I could rename the file as 'traffic_filter_windows.go'. 

`allowNodeToPeerCIDRTraffic()` should not be the single method of the interface. These methods are required outside package "iptables" on Linux Node: 1) NewClient(), 2) Initialize, 3) Reconcile, 4) AllowNodeToPeerCIDRTraffic. If we define an interface and return it in function "NewClient",  all above functions should be defined in the interface, and the OS specific implementations must be privoded. 

We have no such functions in current _windows.go file is because we returned the struct directly, and no other functions except for "NewClient" are called outside "iptables" on Windows .

Reply:
Yes, you need Initialize() at least. NewClient() should not be defined in the interface, but could be in agent_os.go.

For Reconcile() I was thinking you might convert the trafficFilter interface to iptablesClient in node_route_controller_windows.go, and call iptablesClient.reconcile() directly. But if you think that is strange, and not many other funcs, we might add Reconcile() to the interface.